src/lib.rs

```diff
@@ -633,6 +635,7 @@ impl bytes_rs::BufMut for CircBuf {
 mod tests {
     extern crate vecio;
 
+    #[cfg(unix)]
```

Comment:
Just curious, what's the motivation for adding this configuration option, do these tests not work on Windows? I only have a Unix machine myself so I'm not able to check locally.

Reply:
Yes I was play.. working on window, `vecio` implement [`Rawv` ](https://docs.rs/vecio/latest/vecio/trait.Rawv.html#implementors) for every `T: AsRawFd` and [`AsRawFd`](https://doc.rust-lang.org/std/os/unix/io/trait.AsRawFd.html) is only implemented for unix in std. Also `vecio` is anyway unix specific api.

Reply:
thus, possibly we can remove the use of vecio in a next PR, since STD have [`read_vectored`](https://doc.rust-lang.org/nightly/std/io/trait.Read.html#method.read_vectored). But I make PR step by step.

Reply:
Gotcha, thanks for the explanation!